tests/META-INF/MANIFEST.MF

```diff
@@ -7,7 +7,8 @@ Bundle-Version: 0.1.0
 Bundle-RequiredExecutionEnvironment: JavaSE-11
 Require-Bundle: org.junit.jupiter.api;bundle-version="[5.8.0,6.0.0)",
  org.eclipse.ui,
- org.hamcrest.library;bundle-version="[1.3.0,2.2)"
+ org.hamcrest.library;bundle-version="[1.3.0,2.2)",
+ io.openliberty.tools.eclipse.ui
```

Comment:
We're mixing require-bundle and import-package?  Why not one or the other?

Reply:
oh - i see...ok i will fix that, that seems to have gotten mixed up in the rename rebase i did

Reply:
got mixed up in the rename rebase ... i will fix